packages/verified-fetch/src/index.ts

```diff
@@ -189,6 +194,24 @@
  *   }
  * })
  * ```
+ * ### Custom Hashers
+ *
+ * By default, `@helia/verified-fetch` supports `sha256`, `sha512`, and `identity` hashers.
```

Comment:
I would instead, or in addition, point to helia defaults.. that could help if this ever gets out of date